renderer/models/QuestionList.tsx

```diff
@@ -28,6 +37,10 @@ export class QuestionList {
     getQuestionsLeft(): number {
```

Comment:
the name suggests than you are returning left questions, not the number of left questions. Maby sth like getQuestionsLeftAmount()